feat: DIA-1402: V1-Submit Prompt auto-refinement job

robot-ci-heartex opened the pull request:
Hi @robot-ci-heartex!

This PR was automaticaly generated via Follow Merge. 
Please ensure that all linked upstream Pull Requests are merged before proceeding with this one.

- https://github.com/HumanSignal/label-studio-sdk/pull/322 linked by [Workflow Run](https://github.com/HumanSignal/label-studio/actions/runs/11042151014)

matt-bernstein committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/11042151014

fern-api[bot] committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/11131172115

fern-api[bot] committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/11131866575

fern-api[bot] committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/11132380340

matt-bernstein commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/11132510599)
> Successfully merged:  create mode 100644 web/libs/editor/src/stores/types.d.ts

matt-bernstein committed:
Merge branch 'develop' into 'fb-dia-1402'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/11132510599

fern-api[bot] committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/11133653423

robot-ci-heartex committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/11134831525
